src/mono/wasm/build/README.md

```diff
@@ -12,24 +12,24 @@ various properties before the target `WasmBuildApp` gets executed.
 to do that.
 
 - By default, the `WasmLoadAssembliesAndReferences` task is not run, and
-the specified `@(WasmAssembliesToBundle)` are directly passed to
+the specified `@(WasmAssembly)` are directly passed to
```

Comment:
WasmAOTAssembly? WasmAssembly feels very non-specific.

Reply:
they are not AOT specific